target/linux/ath79/image/mikrotik.mk

```diff
@@ -35,3 +35,19 @@ define Device/mikrotik_routerboard-wap-g-5hact2hnd
   SUPPORTED_DEVICES += rb-wapg-5hact2hnd
 endef
 TARGET_DEVICES += mikrotik_routerboard-wap-g-5hact2hnd
+
+define Device/mikrotik_sxt5n
+  $(Device/mikrotik)
+  SOC := ar9344
+	DEVICE_MODEL := SXT5N platform
+  IMAGE/sysupgrade.bin = append-kernel | kernel2minor -s 2048 -e -c | \
+	sysupgrade-tar kernel=$$$$@ | append-metadata
+  DEVICE_PACKAGES += nand-utils rssileds kmod-gpio-beeper
+  SUPPORTED_DEVICES += rb-sxt5n
```

Comment:
Even if it may be redundant, I'd very much prefer to keep SUPPORTED_DEVICES to the actual devices.

Reply:
Fine with that.